src/components/interfaces/HMI_API.xml

```diff
@@ -3981,6 +3981,14 @@
     </function>
     <function name="ActivateApp" messagetype="response">
     </function>
+    <function name="CloseApplication" messagetype="request">
+      <description>Request from the application to exit the foreground.</description>
+      <param name="appID" type="Integer" mandatory="true">
```

Comment:
Is this the hmi app id or the policy app id?

Reply:
HMI app ID. It's an Integer, not a String